docs/readthedocs/use/osmInputData.md

```diff
@@ -0,0 +1,122 @@
+# Openstreetmap Input Data
+
+## How to get Osm Input Data
+### Overpass
+OSM data can be easily collected by using [overpass](https://overpass-turbo.eu/) but needs to be converted into .pbf format before using OsmoGrid.
+
+#### Remarks for using overpass
+We require the following information in your input data:
+  - Boundaries,
+  - Buildings,
+  - Streets,
+  - Landuse,
+  - Substations.
+
+Thus filtering for nodes, ways and relations for these categories is required.
+
+Since even small areas can result in huge grid data, we encourage to limit the export to limited areas. This can be done e.g. by boundary boxes or based on relations. For more options, please check [Overpass API User's Manual](https://dev.overpass-api.de/overpass-doc/en/).
+
+#### Example query based on bounding box 
+
+This example will return the osm data for the area within given coordinates. They usually follow the standard format of: min longitude, min latitude, max longitude, max latitude. 
+
+```
+[out:xml][timeout:10][bbox:52.0262,7.3008,52.03672,7.31806];
+// gather results
+(
+
+  node["boundary"];
+  way["boundary"];
+  relation["boundary"="administrative"]["admin_level"~"^(6|7|8|9)$"]({{searcharea}});
```

Comment:
Executing this query throws an error because `searcharea` is not defined. You could use `{{bbox}}` instead since you define it above anyways.

Reply:
Good catch!